app/models/manageiq/providers/azure/inventory_collection_default/cloud_manager.rb

```diff
@@ -60,6 +60,54 @@ def key_pairs(extra_attributes = {})
       super(attributes.merge!(extra_attributes))
     end
 
+    def vm_and_template_labels(extra_attributes = {})
```

Comment:
@Ladas @juliancheal  these collections are starting to pile up in multiple providers, do you see a way to share them?
(I'm fine with duplicating for gaprindashvili & fine, then refactoring)

Reply:
to clarify, I'm not sure sharing vm_and_template_labels/taggings between 2 providers is a big win;
I'm more thinking of a "polymorphic" anything-custom-attributes, anything-taggings, something like these:
https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/5ec31943f8/app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb#L359-L400

Reply:
yeah, we should probably try to unify these next, from what I saw, each provider had it slightly different though

Reply:
Yeah sounds good to me.